middleware/context-builder.js

```diff
@@ -8,9 +8,17 @@ module.exports = function contextBuilder (req, res, next) {
   // Attach i18n object to request so any route handler can use it if needed
   req.i18n = i18n
 
+  const localized = i18n.website[req.language]
+
   // Page titles, descriptions, etc
-  const page = i18n.website[req.language].pages[req.path] || {}
-  page.path = req.path
+  let page = Object.assign({
+  	title: `${localized._404.page_not_found} | Electron`,
```

Comment:
There will be some cases where the page is found doesn't have a title in locale.yml. Instead of setting the title to 404 there, the title should just default to something like 'Electron'. Does that make sense?

Reply:
Ok